tests/unit/test_pool.py

```diff
@@ -241,25 +241,36 @@ def test_spans_bind_get(self):
 
         with trace_call("pool.Get", SESSIONS[0]) as span:
             pool.get()
-            wantEventNames = [
-                "Acquiring session",
-                "Waiting for a session to become available",
-                "Acquired session",
-            ]
-            self.assertSpanEvents("pool.Get", wantEventNames, span)
 
-        # Check for the overall spans too.
+        span_list = self.get_finished_spans()
+        got_span_names = [span.name for span in span_list]
+        want_span_names = ["CloudSpanner.FixedPool.BatchCreateSessions", "pool.Get"]
+        print("got", got_span_names)
+        assert got_span_names == want_span_names
+
+        attrs = TestFixedSizePool.BASE_ATTRIBUTES.copy()
+        attrs["db.instance"] = ""
```

Comment:
Why is this "" here, though we are passing "name" as the database name above?
Looks like this is because of mock class?

Reply:
Well we are using the mock _Database for which we have to then add an observability_options method which is fallacious because that doesn't even actually test our function, but let me add it.

Reply:
Done, thanks!